src/hdmf_zarr/nwb.py

```diff
@@ -63,5 +64,30 @@ def export(self, **kwargs):
             kwargs['container'] = nwbfile
             super().export(**kwargs)
 
+        @staticmethod
+        @docval({'name': 'path',
+                 'type': (str, Path, *SUPPORTED_ZARR_STORES),
+                 'doc': 'the path to the Zarr file or a supported Zarr store'},
+                is_method=False)
```

Comment:
Hi, Oliver, I am curios about what this does? (`is_method`)

A naive reading of mine indicates that this is a method?

Reply:
Maybe related to static method? documentation display? throwing guesses here.

Reply:
`is_method=False` tells docval that there is no `self` or `cls` parameter to allow us to use docval with static functions and functions defined outside of the scope of classes.